package.json

```diff
@@ -4,7 +4,11 @@
   "license": "UNLICENSED",
   "description": "Meetup Web Mocks",
   "author": "Meetup",
-  "engine": "node >=6.5.0",
+  "engineStrict": true,
+  "engines": {
+    "node": "^10.16.3",
```

Comment:
In the future, unless there are code changes, let's specify compatibility with multiple versions, e.g.

```json
"node": "8 || 10",
```

so that we don't have to wait to merge this change - we could still do it now, but we're close to merging everything else so it's up to you